templates/step/edit.html.ep

```diff
@@ -140,15 +148,19 @@
                             </p>
                             <p>
                                 <!-- lists the specific properties -->
-                                <input type="checkbox" name="workaround" id="property_workaround"
-                                        %= 'checked' if $properties->{workaround}
-                                        >
+                                <input type="checkbox" name="workaround" id="property_workaround" >
                                 <label for="property_workaround">workaround</label>
                                 <%= help_popover('Help for the workaround property' =>
                                     'If a needle with the workaround property matches the match result is recorded as
                                     a \'soft-fail\'. It is suggested to mark the reason for this choice in either the
-                                    name of the needle (e.g. my-name-bsc1234-20161224) or with special tags')
+                                    name of the needle (e.g. my-name-bsc1234-20161224) or with special tags and
+                                    input the reason description in \'Description\' textbox.')
```

Comment:
This is confusing. When I first read I thought 'special tags' and the 'description' need to be used together. It would be best to remove the old suggestion and only suggest using the description anymore.

Reply:
I will fix it.